libs/sdk-ui-charts/src/charts/headline/Headline.tsx

```diff
@@ -81,8 +84,34 @@ export const Headline = (props: IHeadlineProps) => {
 };
 
 export function RenderHeadline(props: IHeadlineProps): JSX.Element {
-    invariant(props.primaryMeasure, "The property primaryMeasure must be specified.");
-    return <CoreHeadline {...toCoreHeadlineProps(props)} />;
+    const { backend, workspace, primaryMeasure } = props;
+    invariant(primaryMeasure, "The property primaryMeasure must be specified.");
+
+    const [isEnableNewHeadline, setEnableNewHeadline] = useState<boolean>();
+
+    // TODO - this block should be removed when removing FF enableNewHeadline.
+    useEffect(() => {
+        if (backend && workspace) {
+            backend
+                .workspace(workspace)
+                .settings()
+                .getSettingsForCurrentUser()
+                .then(
+                    (featureFlags) => {
+                        // the logical or '!!' operator using to ensure
+                        // that the isEnableNewHeadline variable is either true or false (not undefined or null)
```

Comment:
I think that this comment does not add much value. I would remove it.

Reply:
removed!